packages/common/test/events/_internal/getEventGroups.test.ts

```diff
@@ -1,4 +1,4 @@
-import { getEventGroups } from "../../../src/events/_internal/getEventGroups";
+import { getEventGroups } from "../../../src/events/getEventGroups";
```

Comment:
Minor, but since we moved the `getEventGroups.ts` module, lets also move `getEventGroups.test.ts`. 

`packages/common/test/events/getEventGroups.test.ts`